pypeapp/pypeLauncher.py

```diff
@@ -40,17 +38,23 @@ def print_info(self):
             infos.append(("Using Muster at",
                           os.environ.get("MUSTER_REST_URL")))
 
-        if host:
-            infos.append(("Logging to mongodb", "{}/{}".format(
-                host, database)))
-            if port:
-                infos.append(("  - port", port))
-            if username:
-                infos.append(("  - user", username))
-            if collection:
-                infos.append(("  - collection", collection))
-            if auth_db:
-                infos.append(("  - auth source", auth_db))
+        if components["host"]:
+            infos.append((
```

Comment:
`if os.environ.get("PYPE_LOG_MONGO_DB"):` needs to be here, otherwise it will crash on missing key once the variable is removed from config

Reply:
Nice sport. Done